webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -73,8 +77,39 @@ export class GenericModal extends React.PureComponent<Props, State> {
         };
     }
 
+    componentDidMount(): void {
+        const {focusableElement} = this.props;
+        if (focusableElement && focusableElement.current) {
+            setTimeout(() => {
+                document.dispatchEvent(
+                    new CustomEvent(CUSTOM_FOCUS_EVENT, {
+                        detail: {
+                            target: focusableElement.current,
+                            keyboardOnly: false,
```

Comment:
Does this cause the visible a11y styling to disappear when the user opens/closed the modal with the keyboard?